searchcore/src/vespa/searchcore/proton/matching/matcher.cpp

```diff
@@ -281,16 +280,33 @@ Matcher::match(const SearchRequest &request,
         reply = std::move(result->_reply);
         SearchReply::Coverage & coverage = reply->coverage;
         if (wasLimited) {
+            LOG(debug, "was limited, degraded from match phase");
             coverage.degradeMatchPhase();
         }
         if (my_stats.softDoomed()) {
+            LOG(debug, "soft doomed, degraded from timeout");
             coverage.degradeTimeout();
         }
-        coverage.setActive(metaStore.getNumActiveLids());
+        uint32_t numActiveLids = metaStore.getNumActiveLids();
+        // note: this is actually totalSpace+1, since 0 is reserved
```

Comment:
this comment makes it more confusing, since the off-by-one does not come from document 0, but from the fact that one number is a limit (end marker) and the other is a size (end - 1)